handlers/get_metadata_handler.py

```diff
@@ -0,0 +1,31 @@
+import json
+from typing import Union
+
+from aws_lambda_context import LambdaContext
+
+from usecases.get_metadata import GetMetadata
+from utils.wrappers.aws.api_gateway import ApiGateway, ApiGatewayEvent
+
+
+@ApiGateway()
+def handle(
+    event: ApiGatewayEvent,
+    _context: LambdaContext,
+    get_metadata: Union[GetMetadata, None] = None
+) -> dict:
+
+    get_metadata = get_metadata or GetMetadata()
+    type = event.path['type']
+    hash = event.path['hash']
+    response = get_metadata.get(type, hash)
+
+    if response is None:
+        raise Exception('not_found')
```

Comment:
Should we raise an exception in the lambda? Can't it be confused with an unhandled exception? I feel the lambda will return 500 instead of the 404 expected in the design.